client/src/components/SolutionViewer/SolutionViewer.tsx

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import { Panel } from '@freecodecamp/react-bootstrap';
+import Prism from 'prismjs';
+import React from 'react';
+
+interface solutionviewerProps {
+  challengeFiles: [];
+  solution: string;
+}
+
+function SolutionViewer({
+  challengeFiles,
+  solution = '// The solution is not available for this project'
+}: solutionviewerProps): JSX.Element {
+  return (
+    <>
+      {challengeFiles?.length ? (
+        challengeFiles.map((challengeFile: string) => (
+          <Panel
+            bsStyle='primary'
+            className='solution-viewer'
+            key={challengeFile}
+          >
+            <Panel.Heading>{challengeFile.toUpperCase()}</Panel.Heading>
+
+            <Panel.Body>
+              <pre>
+                <code
+                  dangerouslySetInnerHTML={{
+                    __html: Prism.highlight(
+                      challengeFile.trim(),
+                      Prism.languages[challengeFile],
```

Comment:
```suggestion
                      challengeFile.contents.trim(),
                      Prism.languages[challengeFile.ext],
```
Correct property access.